### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TheIceCreamTroll**
## Describe your changes

~~- Some out-of-date seasons in the readme/exceptions have been updated (aside from Water Seven, which is addressed by #32)~~
~~- Removed some excess nfo files in a completed season~~
- Reference to column P in the [One Pace spreadsheet](https://docs.google.com/spreadsheets/d/1HQRMJgu_zArp-sLnvFMDzOyjdsht87eFLECxMK858lA/edit#gid=0) has also been ~~removed~~ reworded, as the column no longer exists as described.
- Quick readme editing pass

## Checklist for submitting new .nfo files
- [x] I have removed the old .nfo files that are being replaced
- [ ] I have validated that the new .nfo file works correctly in Plex
- [x] I have updated the README, fixing the list of current missing episodes
- [x] I have modified exceptions.json, removing any exceptions no longer required

### CODE REVIEW

**Review by cawnj — CHANGES_REQUESTED** [review]
Hey @TheIceCreamTroll , thanks for the change again!

Column P in the Episode Spreadsheet has now changed to Column Q. And that explains in depth which episodes we need to have present here.
It seems that you didn't spot the info here, as all the episodes and exceptions - from what I can tell  - are as expected, except for Water Seven of course which is still in review.

Nice catch on all the other errors in the README though, those changes look good to me! Just the episode related changes need to be dropped.
I also added some comments explaining how the exceptions work a bit more and why we need them for certain episodes/arcs

File: README.md [hunk 1]
```diff
@@ -61,15 +61,14 @@ Download all the One Pace episodes you wish to add and place them in their respe
...
 Current missing episodes:
-- Season 14: 121-130
-- Season 16: 171-195, 207
+- Season 14: 122-130
+- Season 16: 171-195
 - Season 18: 260-263
-- Season 24: 453-456
 
-See the [One Pace Episode Spreadsheet](https://docs.google.com/spreadsheets/d/1HQRMJgu_zArp-sLnvFMDzOyjdsht87eFLECxMK858lA/) for a more up to date information on what episodes are avaliable. Check column P for episodes that are still to be completed.
+See the [One Pace Episode Spreadsheet](https://docs.google.com/spreadsheets/d/1HQRMJgu_zArp-sLnvFMDzOyjdsht87eFLECxMK858lA/) for a more up-to-date information on what episodes are available.
```
[the remark this answers is not in the document]
Comment (TheIceCreamTroll) [reply README.md:71]: Rephrasing this, as the column doesn't really show which episodes are missing (Impl Down is complete, yet still lists OG episodes), but instead shows which original episodes are needed for a "complete" watch order

**Comment by TheIceCreamTroll** [comment]
Thanks for the feedback. I had assumed on first glance that the exceptions file was just for existing One Pace episodes, and tunnel-visioned on the individual season pages in the spreadsheet, which don't include the original episode info in column Q. Sorry for the bother there, and thanks for the explanations in exceptions.json.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (cawnj) [reply README.md]: Seems reasonable enough to me!

**Comment by cawnj** [comment]
> Thanks for the feedback. I had assumed on first glance that the exceptions file was just for existing One Pace episodes, and tunnel-visioned on the individual season pages in the spreadsheet, which don't include the original episode info in column Q. Sorry for the bother there, and thanks for the explanations in exceptions.json.

Hey no problem at all, glad to help. I'm definitely guilty of tunnel-visioning on problems myself!
Hopefully you at least have a better understanding now of how all the exceptions and weird edge cases work :)

**Review by cawnj — APPROVED** [review]
+1 on the README edits, looks good to me! Ty for the change 🙏

**Comment by TheIceCreamTroll** [comment]
Happy to help! Now that I'm looking at column Q, it seems like Wano has a bunch of original episodes listed (968-1085) that don't yet have nfo files. Would a PR for that be wanted? Happy to work on it if so.

**Comment by cawnj** [comment]
@TheIceCreamTroll Yep I'm aware of that one actually. When this project started, Wano was still ongoing so those nfo's hadn't been made up yet.

Funny you mention it because I had just started work on it recently, I was doing it on my laptop in batches due to not having enough disk space, which was super tedious so I had stopped. I think I'll just switch over to my desktop to do it there, or mount my NAS as a network drive, since I'll actually have the disk space then to do it all in one go!!

If you'd like to tackle it yourself and save me the trouble of finishing it, feel free to! Otherwise I'll probably get back around to it sometime soon :)

**Comment by TheIceCreamTroll** [comment]
@cawnj I can give it a shot. Do you have any recommended settings for tinyMediaManager (esp metadata scrapers)? The nfo's I export contain more info than the ones in this project, and ideally those wouldn't be removed by hand

**Comment by SpykerNZ** [comment]
Some nice work here, thanks for the README tweaks.

I'm only watching the earlier episodes, so haven't been maintaining the Wano arc - just existing arcs when I have the time. Would be great to have that included though!

**Comment by cawnj** [comment]
@SpykerNZ I'm in a similar situation, currently on Enies Lobby, so it hasn't been high priority for me at all get to get Wano up to scratch.

@TheIceCreamTroll In the right-click menu there's the option "Scrape metadata for selected TV show(s)", and that gives you the option to specify which fields to scrape info for. That should be what you're looking for to ensure the nfo's have the bare minimum info needed.

Aside from that, I think the best way is to start out with One Piece season numbers and episodes, bulk scrape the data and then bulk edit the episodes to match the One Pace season numbers here.

**Comment by TheIceCreamTroll** [comment]
Threw together a python script last night that helps get the nfo files to look like the rest of the one pace ones. Currently checking Plex to make sure they work, will prob have a PR ready sometime later today

**Comment by TheIceCreamTroll** [comment]
Managed to python my way out of doing the exceptions.json edits by hand, so the PR is ready early! #34

**Pull request merged**